src/libical/icalduration.c

```diff
@@ -294,7 +294,7 @@ int icaldurationtype_is_null_duration(struct icaldurationtype d)
     }
 }
 
-/* in icalvalue_new_from_string_with_error, we should not call
```

Comment:
should this be a doxygen comment?

Reply:
It looks to me like an internal implementation comment not a docstring, since it only refers to how `icalvalue_new_from_string_with_error` should be implemented. Maybe it should be over there instead?

Reply:
correct. not a docstring we should have in the API documentation.